config/checker_guideline_map.json

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
I think it would be great to create a short documentation about this file maybe under https://github.com/Ericsson/codechecker/tree/master/docs/analyzer.